Sources/ViewControllers/Settings/SettingsTableViewController.swift

```diff
@@ -29,6 +29,9 @@ class SettingsTableViewController: UITableViewController {
         }
     }
 
+    @IBAction func didTapOpenBeautyFacts(_ sender: UIButton) {
+        UIApplication.shared.openURL(NSURL(string: "itms-apps://itunes.apple.com/us/app/open-beauty-facts/id1122926380?mt=8")! as URL)
```

Comment:
Why using NSURL instead URL(string: "Your URL:)?

If you want to avoid force cast. I would suggest to use guard let url = URL(string: "YOUR URL") else { return } 

It will be also appreciate yo use URLs.swift instead hard coded string.

Reply:
Thank you for the review @roger-tan 
I understood why the requested changes are required, and changed them.

Thank you!